app/components/form/fields/RadioField.tsx

```diff
@@ -6,7 +6,7 @@
  * Copyright Oxide Computer Company
  */
 import cn from 'classnames'
-import { useId, type default as React } from 'react'
+import { useId } from 'react'
```

Comment:
I had changed these to this ugly `type default` thing in #2713 to satisfy oxlint complains about type imports, but it turns out that when `React` is used as just a type, it's actually global already and we don't have import it at all.